Loop54.Shared/Model/Request/GetEntitiesByAttributeRequest.cs

```diff
@@ -11,40 +14,71 @@ public class GetEntitiesByAttributeRequest : Request
     {
         public GetEntitiesByAttributeRequest(string attributeName, string attributeValue)
         {
-            Attribute.Name = attributeName;
-            Attribute.Value = attributeValue;
+            Attribute = new AttributeNameValuePairSingle()
+            {
+                Name = attributeName,
+                Value = attributeValue
+            };
         }
-        
+
+        public GetEntitiesByAttributeRequest(string attributeName, string[] attributeValue)
+        {
+            Attribute = new AttributeNameValuePairMultiple()
+            {
+                Name = attributeName,
+                Value = attributeValue
+            };
+        }
+
         /// <summary>
         /// The attribute name-value-pair to find entities connected to. Note: this attribute needs 
         /// to be indexed in the engine. See the endpoint /getIndexedAttributes.
         /// </summary>
-        public AttributeNameValuePair Attribute { get; set; } = new AttributeNameValuePair();
+        public AttributeNameValuePair Attribute { get; set; }
 
         /// <summary>
         /// Parameters for specifying which results to retrieve and how to format them.
         /// </summary>
         public EntityCollectionParameters ResultsOptions { get; set; } = new EntityCollectionParameters();
 
+        public abstract class AttributeNameValuePair
```

Comment:
Looks a bit messy but all I did was extract Name to a common base class, and added 2 new classes for the single and multi value properties.